content/en/security_platform/notifications/variables.md

```diff
@@ -0,0 +1,263 @@
+
+### Epoch
+
+Epoch template variables create a human-readable string or math-friendly number within a notification. For example, use values such as `first_seen`, `last_seen`, or `timestamp` (in milliseconds) within a function to receive a readable string in a notification. For example:
+
+```
+{{eval "first_seen_epoch-15*60*1000"}}
+```
+
+For more information on the `eval` function, see [Template Variable Evaluation][2].
+
+### Local time
+
+Use the `local_time` function to add another date in your notification in the time zone of your choice. This function transforms a date into its local time: `{{local_time 'time_variable' 'timezone'}}`. 
+
+For example, to add the last triggered time of the signal in the Tokyo time zone in your notification, include the following in the notification message:
+
+```
+{{local_time 'last_triggered_at' 'Asia/Tokyo'}}
+```
+
+The result is displayed in the ISO 8601 format: `yyyy-MM-dd HH:mm:ss±HH:mm`, for example, `2021-05-31 23:43:27+09:00`. See the [list of TZ database time zones][3], specifically the `TZ database name` column, to see the list of available time zone values.
+
+## Attribute variables
+
+Use attribute variables to customize signal notifications with specific information about the triggered signal. 
+
+To see a signal’s list of event attributes, click **JSON** at the bottom of the **Overview** tab in the signal’s side panel. Use the following syntax to add these event attributes in your rule notifications: `{{@attribute}}`. To access inner keys of the event attributes, use JSON dot notation, for example, `{{@attribute.inner_key}})`.
+
+The following is an example JSON object with event attributes that may be associated with a security signal:
+
+{{< tabs >}}
+{{% tab "Cloud SIEM" %}}
+
+```json
+{
+  "network": {
+    "client": {
+      "ip": "1.2.3.4"
+    }
+  },
+  "usr": {
+    "id": "user@domain.com"
+  },
+  "evt": {
+    "category": "authentication",
+    "outcome": "success"
+  },
+  "used_mfa": "false"
+}
+```
+
+If you use the following in the “Say what’s happening” section:
+
+```
+{{@usr.id}} just logged in without MFA from {{@network.client.ip}}.
+```
+This is what the notification message looks like:
+
+```
+user@domain.com just logged in without MFA from 1.2.3.4.
+```
+
+{{% /tab %}}
+
+{{% tab "Application Security Monitoring" %}}
+
+```json
+{
+  "attributes":{
+    "title":"Security scanner detected",
+    "http":{
+      "url":"http://www.example.com"
+    },
+    "rule":{
+      "detectionMethod":"threshold",
+      "name":"Your rule name"
+    },
+    "events_matched":2,
+    "first_seen":"2022-01-26T13:23:33.000Z",
+    "last_seen":"2022-01-27T04:01:57.000Z"
+  },
+  "groupByPaths":[
+    "service"
+  ]
+}
+```
+
+If you use the following in the Say What's Happening section:
+
+```
+Real routes targeted for {{@service}}.
+```
+
+The notification shows the service name in the message as follows:
+
+```
+Real routes targeted for your_service_name.
+```
+
+{{% /tab %}}
```

Comment:
We need to swap the content of the tabs

Reply:
Hey @btoueg! The content was taken directly from the specific product detection rule pages, which was what PMs had put in:

From Cloud SIEM's log detection rule page: https://docs.datadoghq.com/security_platform/cloud_siem/log_detection_rules?tab=threshold#say-whats-happening 

From ASM's custom detection page: https://docs.datadoghq.com/security_platform/application_security/custom_rules/#template-variables

Reply:
Ok my bad, I'll see if I can provide better suggestion later